nix/solidity-package.sh

```diff
@@ -20,7 +20,10 @@ mkdir lib
 echo "$LIBSCRIPT" > setup.sh
 source setup.sh
 export DAPP_LIB=lib
-dapp2-test-hevm
+
+if [ "$doCheck" == 1 ]; then
```

Comment:
Just to confirm: `true` is translated to 1? Do you know where this happens?

Reply:
Yes, because `doCheck` gets set as an environment variable it's interpolated with `toString` which is `"1"` for `true`.